config/routes.rb

```diff
@@ -67,7 +67,6 @@
   match '/test/session' => 'test#new_session', via: :get
   match '/test/claim' => 'test#new_claim', via: :get
   match '/test/link' => 'test#new_link', via: :get
-  match '/test/source' => 'test#new_source', via: :get
```

Comment:
We want to keep creating sources from the test controller.

Reply:
will back it if agreed to change the logic to create Source instead of ProjectSource